Update first release procedure (#2640)
nf-core/website: sites/docs/src/content/docs/tutorials/adding_a_pipeline/first_release.md

Ok - now the tough bit - does your workflow stand up to the scrutiny of the nf-core team?!
Not to worry, we're a friendly bunch.

To start a review for the initial release of your pipeline:

> 1. Create a new branch named `TEMPLATE_<TOOLS_VERSION>`, based on the version of `TEMPLATE` from the `dev `branch you plan to merge into `master`.

Reviewer: I never create this branch, I just make a the PR directly to the `TEMPLATE`, why would I need the versioned branch, @maxulysse ?

To get your pipeline reviewed for its initial release, do the following:

> 1. Create a new branch based on `TEMPLATE`, called `TEMPLATE_<TOOLS_VERSION>` from the latest tools sync that was merged in `dev`.

Reviewer 2: ```suggestion
1. Create a new branch based on `TEMPLATE`, called `TEMPLATE_<TOOLS_VERSION>` from the version of the TEMPLATE (i.e., latest tools sync) that is in the `dev` branch you wish merge into `master`.
```

> 1. Create a new branch based on `TEMPLATE`, called `TEMPLATE_<TOOLS_VERSION>` from the version of the TEMPLATE (i.e., latest tools sync) that is in the `dev` branch you wish merge into `master`.

Author: In case people would inadvertently merge it

> 1. Create a new branch based on `TEMPLATE`, called `TEMPLATE_<TOOLS_VERSION>` from the version of the TEMPLATE (i.e., latest tools sync) that is in the `dev` branch you wish merge into `master`.

Reviewer: they can't, thanks to branch protection

To get your pipeline reviewed for its initial release, do the following:

1. Create a new branch based on `TEMPLATE`, called `TEMPLATE_<TOOLS_VERSION>` from the latest tools sync that was merged in `dev`.

>    The commit message from @nf-core-bot will help you figure what was the tools versions used.

Reviewer 3: Would an example of a commit message help explain this?

Author: good idea

> To get your pipeline reviewed for its initial release, do the following:

Reviewer 3: ```suggestion
To start a review for your pipeline initial release:
```

> 1. Create a new branch based on `TEMPLATE`, called `TEMPLATE_<TOOLS_VERSION>` from the version of the TEMPLATE (i.e., latest tools sync) that is in the `dev` branch you wish merge into `master`.

Reviewer 3: ```suggestion
1. Create a new branch based on `TEMPLATE`, called `TEMPLATE_<TOOLS_VERSION>` from the version of the TEMPLATE (i.e., latest tools sync) that is in the `dev` branch you wish to merge into `master`.
```

>    The commit message from @nf-core-bot will help you figure what was the tools versions used.

Reviewer 3: ```suggestion
   The commit message from @nf-core-bot will help you figure out what tool versions were used.
```

>    For example, if this commit message `Template update for nf-core/tools version 1.13` is the one that used for a tools sync,
>    the branch should be called `TEMPLATE_1.13`.

Reviewer 3: ```suggestion
   For example, if the tools sync commit message was `Template update for nf-core/tools version 1.13`, call the branch `TEMPLATE_1.13`.
```
